packages/features/form-builder/FormBuilder.tsx

```diff
@@ -438,9 +441,11 @@ function FieldEditDialog({
   const variantsConfig = fieldForm.watch("variantsConfig");
 
   const fieldTypes = Object.values(fieldTypesConfigMap);
+  const [firstRender, setFirstRender] = useState(true);
 
   return (
-    <Dialog open={dialog.isOpen} onOpenChange={onOpenChange}>
+    <Dialog open={dialog.isOpen} onOpenChange={onOpenChange} modal={false}>
+      <div className="fadeIn fixed inset-0 z-50 bg-neutral-800 bg-opacity-70 transition-opacity dark:bg-opacity-70" />
```

Comment:
Why is this changed?

Reply:
Hey, when modal={true} is set to true (default), interaction with outside elements will be disabled, and only the dialog content will be visible to screen readers. For that reason, we can't make text a link in the editor, as it opens another tooltip. So, I set modal to false and externally add the dialog styles that are present by default, because these styles are removed when modal is set to false.

Reply:
This seems hackish. Let's figure out a better way to handle it?

Hackish because we are using the internal details of dialog to make it look like it.

Reply:
Yeah, update it